CONTRIBUTING.md

```diff
@@ -50,10 +52,10 @@ $ bundle exec rake
 ```
 
 However, this can take a while as there are over 1000 links that it checks.
```

Comment:
Shouldn't lines 48-50 - `bundle exec rake` be removed as well?

Reply:
I'm not sure if there will be a `Rakefile` later on (@Carlgo11 ?) Currently this command results in an error message, I'll give you that.

Reply:
No rake plans except for the garden 👨‍🌾